src/cli-validator/index.js

```diff
@@ -6,7 +6,8 @@
 require('./utils/checkVersion')('8.9.0');
 require('./utils/updateNotifier');
 
-const program = require('commander');
+// include the modified commander module
+const program = require('./utils/unsupportedOptionError.js');
```

Comment:
The only change we've made is for unsupported options, but conceptually that's not what we're importing. I think it's important to know that we're bring in the commander module, so I suggest changing your module name to something like `modified-commander`.

```suggestion
const program = require('./utils/modified-commander.js');
```